include/nodemanagerbase.h

```diff
@@ -33,7 +33,10 @@ class NodeManagerConfig
 class NodeManagerBase: public NodeManagerConfig
 {
 public:
-    NodeManagerBase( const char* uri, bool sth, int hashtablesize );
+    NodeManagerBase(
+            const char* sNamespaceUri,
+            bool        firesEvents = OpcUa_False,
+            OpcUa_Int32 nHashTableSize = 9973 );
```

Comment:
what's the magic number? 
Maybe name it with a
`const static OpcUa_Int32 g_sFaxNumberOfTheBeast = 9973`
and so the param list could be a more readable
`OpcUa_Int32 nHashTableSize = g_sFaxNumberOfTheBeast`

Reply:
@ben-farnham "hash table size", to my mind, is a common term in data structures, so I'm not particularly willing to divert from standard terminology. 
9973 is an arbitrary chosen prime number, the rationale for which is common knowledge.